src/main/java/io/github/cottonmc/cotton/gui/widget/WGridPanel.java

```diff
@@ -6,23 +6,82 @@
  * A panel that positions children in a grid.
  */
 public class WGridPanel extends WPanelWithInsets {
+
+	private static final int DEFAULT_GRID_SIZE = 18;
 	/**
 	 * The grid size in pixels.
 	 * Defaults to 18, which is the size of one item slot.
 	 */
-	protected int grid = 18;
+	protected int grid;
```

Comment:
No need for these changes either

Reply:
Since we some users might want to specify a custom gap without giving a custom grid size, we end up with having to specify the default grid size in multiple places, and for stuff like that i usually have a constant.

Reply:
In that case, the default grid size can be a public and documented constant and also the default value of the `grid` field.